### PR TITLE
Feature/bordereau

### PR DESCRIPTION
**Pull request by meletios**

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java [hunk 1]
```diff
@@ -34,231 +35,262 @@
...
+            .orElse(new InsuranceBillingDTO(memberId, Money.of(0L, "SEK")));
+    }
+
+    @Override
+    public List<InsuranceBordereauDTO> getMonthlyBordereau(YearMonth period,
+        ProductTypes productType) {
+        List<ProductEntity> productList = productRepository
+            .findBillingByDate(period.getMonthValue(), period.getYear());
+
+        Stream<InsuranceBillingInfo> filterdList = productList.stream()
```
Comment (hedvigoscar) [inline src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java]: ^filterd^filtered

File: src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java [hunk 2]
```diff
@@ -34,231 +35,262 @@
...
+            .filter(x -> (x.state == ProductStates.SIGNED || x.state == ProductStates.TERMINATED)
```
Comment (hedvigoscar) [inline src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java]: Would it be wise to use `x.state != ProductStates.QUOTE`?

Either way we may run into trouble if we introduce more `ProductState`s. Perhaps a function `boolean isSignedProductState(ProductState state)`?

Comment (meletios) [reply src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java]: Correct, a method feels better. Done!

File: src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java (hunk 1 above)
Comment (meletios) [reply src/main/java/com/hedvig/productPricing/service/service/InsuranceBillingServiceImpl.java]: fixed

**Pull request merged**